services/app-api/utils/types/types.ts

```diff
@@ -61,7 +61,7 @@ export const enum UserRoles {
 // TEMPLATES
 
 export const enum TemplateKeys {
-  MCPAR = "templates/MCPAR-2022-TEMPLATE.xls",
-  MLR = "templates/MLR-2022-TEMPLATE.xls",
-  NAAAR = "templates/NAAAR-2022-TEMPLATE.xls",
+  MCPAR = "templates/MCPAR_Workbook_AddProgramName_2022.xlsx",
+  MLR = "templates/MLR_Workbook_AddProgramName_2022.xlsx",
+  NAAAR = "templates/NAAAR_Workbook_AddProgramName_2022.xlsx",
```

Comment:
What does add program name mean in 6 months? Would it be better to do NAAAR_Workbook_22June2022? Or something like that?

Reply:
Designers settled on `AddProgramName` to indicate to people downloading it that they should modify the file name to include whichever program that specific MCPAR was for. We decided against using detailed dates/versioning in the name because then it would require a code deployment to swap out the template in the event it needs to be changed. 🤷